setup/install.ab

```diff
@@ -112,6 +112,13 @@ main(args) {
         exit(1)
     }
 
+    // Delete the previous symbolic link
+    ${sudo} rm "{bins_folder}/{target}" failed {
+        echo "Failed to remove the previous amber symbol link."
+        echo "Please make sure that root user can access /usr/local/bin directory."
+        exit(1)
+    }
+
```

Comment:
What if this is a fresh install? The computer may not have any symbolic link in the bins directory

Reply:
In that case there is an error about that the file doesn't exist, can be improved with a check. I will do tomorrow.